### ISSUE TITLE
PositionInSet and SizeofSet property is not defined for 'Workbook' and 'Inspect' tab items

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#355]
### Steps to Reproduce

1. Open "Xamarin Workbook"
2. Products window is opened.
3. Navigate to the "wpf "icon and select "create workbook".
4. Navigate to 'Workbook/view inspect' tab item and Observe Inspect properties

### Expected Behavior
PositionInSet and SizeofSet should be properly defined for tab items 
### Actual Behavior
PositionInSet and SizeofSet property is not defined for 'Workbook' and 'Inspect' tab items
Narrator Behavior: Narrator is announcing as 'selected, workbook tab item'
NVDA behavior:NVDA is announcing as 'Workbook tab selected'

### User Impact:
User who rely on screenreader will not not be able to know position and Size of tab items

### Screen-Shots: 
![positionsize_tabitem_inspect](https://user-images.githubusercontent.com/22956130/40723952-947c2a3e-643d-11e8-8d8f-949ef1be7b97.png)
![positionsize_tabitem_narrator](https://user-images.githubusercontent.com/22956130/40723953-94b0b8bc-643d-11e8-85a0-8ea8ffaf0292.png)
![positionsize_tabitem_nvda](https://user-images.githubusercontent.com/22956130/40723954-94e103dc-643d-11e8-8dff-4567ee851f00.png)

### Environment: 
Windows :Windows 10 Pro Version 1803 (OS Build : 17672.1000)
Xamarin Version : 1.4 (master/b3f92f9)
NVDA Version: 2018.1

**Comment by WiproPSAccTest** [comment]
Please add the following labels to the bug:
A11yMAS
Accessibility
Xamarin-Workbooks-Windows
Wipro
MAS4.2.1
